src/types/index.ts

```diff
@@ -1,9 +1,65 @@
 // @ts-ignore
 interface AutoComplete {}
 
-interface StopGroup {}
+interface StopGroup {
+  country_code: string;
+  name: string;
+  longitude: number;
+  latitude: number;
+  type: string;
+  childs: SegmentProviderStop[];
+  serviced?: string;
+  has_been_modified?: boolean;
+  warning?: boolean;
+  is_latest?: boolean;
+}
+
+interface StopCluster {
+  country_code: string;
+  name: string;
+  longitude: number;
+  latitude: number;
+  type: string;
+  childs: string[];
+  unique_name?: string;
+  serviced?: string;
+  has_been_modified?: boolean;
+  warning?: boolean;
+  is_latest?: boolean;
+}
 
-interface StopCluster {}
+type GroundPlacesList = {
+  [key: string]: StopGroup | StopCluster;
+};
+
+interface StopInfos {
+  countryCode: string;
+  name: string;
+  longitude: number;
+  latitude: number;
+  type: string;
+}
+
+interface GroundPlaceGenerated {
```

Comment:
? what is it ?

Reply:
The result coming from the gp-uid package : https://github.com/tictactrip/gp-uid

Output example:
```
[
    {
        id: 'c|FRparis___@u09yc',
        countryCode: 'fr',
        latitude: 49.00443,
        longitude: 2.51703,
        name: 'Paris, Île-de-France, France',
        type: 'cluster'
    }
]
```

Reply:
ok